sdk/keyvault/azure-keyvault-certificates/samples/hello_world.py

```diff
@@ -49,15 +49,15 @@
     # Alternatively, if you would like to use our default policy, don't pass a policy parameter to
     # our certificate creation method
     cert_policy = CertificatePolicy(
+        issuer_name="Self",
```

Comment:
`WellKnownIssuerNames.Self`?

Reply:
added those back in, it appears when I resolved conflicts I got rid of those changes